scipy/optimize/_differentialevolution.py

```diff
@@ -309,8 +372,14 @@ def __init__(self, func, bounds, args=(),
                  strategy='best1bin', maxiter=None, popsize=15,
                  tol=0.01, mutation=(0.5, 1), recombination=0.7, seed=None,
                  maxfun=None, callback=None, disp=False, polish=True,
-                 init='latinhypercube'):
-
+                 init='latinhypercube', pool=None):
+        
+        if pool is None:
+            self.pool = SPool()
```

Comment:
I'm not sure addition of serial pool makes sense. Why not have:

    if pool is None:
        self.pool_map = map
        self.poolsize = 1
    else:
        self.pool_map = pool.map
        self.poolsize = pool.poolsize()